plugins/janus_nosip.c

```diff
@@ -1186,6 +1186,11 @@ static void *janus_nosip_handler(void *data) {
 			const char *msg_sdp_type = json_string_value(json_object_get(generate ? msg->jsep : root, "type"));
 			const char *msg_sdp = json_string_value(json_object_get(generate ? msg->jsep : root, "sdp"));
 			gboolean sdp_update = json_is_true(json_object_get(generate ? msg->jsep : root, "update"));
+			if(!generate) {
+				if(session->media.ready) {
```

Comment:
Why two layers of if when they can be together?

Reply:
single line - ok)

Reply:
force updated